UnityProject/Assets/Scripts/Managers/MatrixManager.cs

```diff
@@ -206,39 +271,56 @@ public struct MatrixInfo {
 
 	private NetworkInstanceId netId;
 
-	public NetworkInstanceId NetId {
-		get { //late init, because server is not yet up during InitMatrices()
-			if ( netId.IsEmpty() || netId == NetworkInstanceId.Invalid ) {
-				netId = getNetId( Matrix );
+	public NetworkInstanceId NetId
+	{
+		get
+		{
+			//late init, because server is not yet up during InitMatrices()
+			if (netId.IsEmpty() || netId == NetworkInstanceId.Invalid)
+			{
+				netId = getNetId(Matrix);
 			}
+
 			return netId;
 		}
 	}
 
 	/// Null object
 	public static readonly MatrixInfo Invalid = new MatrixInfo {Id = -1};
 
-	public override string ToString() {
-		return Equals(Invalid) ? "[Invalid matrix]" 
-		: $"[({Id}){GameObject.name},offset={Offset},pivot={MatrixMove?.Pivot},state={MatrixMove?.State},netId={NetId}]";
+	public override string ToString()
+	{
+		return Equals(Invalid)
+			? "[Invalid matrix]"
+			: $"[({Id}){GameObject.name},offset={Offset},pivot={MatrixMove?.Pivot},state={MatrixMove?.State},netId={NetId}]";
 	}
+
 	///Figuring out netId. NetworkIdentity is located on the pivot (parent) gameObject for MatrixMove-equipped matrices 
-	private static NetworkInstanceId getNetId( Matrix matrix ) {
+	private static NetworkInstanceId getNetId(Matrix matrix)
+	{
 		var netId = NetworkInstanceId.Invalid;
-		if ( !matrix ) {
+		if (!matrix)
+		{
 			return netId;
 		}
+
 		NetworkIdentity component = matrix.gameObject.GetComponentInParent<NetworkIdentity>();
 		NetworkIdentity componentInParent = matrix.gameObject.GetComponentInParent<NetworkIdentity>();
-		if ( component && component.netId != NetworkInstanceId.Invalid && !component.netId.IsEmpty() ) {
+		if (component && component.netId != NetworkInstanceId.Invalid && !component.netId.IsEmpty())
+		{
 			netId = component.netId;
 		}
-		if ( componentInParent && componentInParent.netId != NetworkInstanceId.Invalid && !componentInParent.netId.IsEmpty() ) {
+
+		if (componentInParent && componentInParent.netId != NetworkInstanceId.Invalid && !componentInParent.netId.IsEmpty())
+		{
 			netId = componentInParent.netId;
 		}
-		if ( netId == NetworkInstanceId.Invalid ) {
-			Debug.LogWarning( $"Invalid NetID for matrix {matrix.gameObject.name}!" );
+
+		if (netId == NetworkInstanceId.Invalid)
+		{
+			Debug.LogWarning($"Invalid NetID for matrix {matrix.gameObject.name}!");
 		}
+
 		return netId;
 	}
-}
+}
```

Comment:
ow, 80 extra lines

Reply:
those are lines of progress

Reply:
nope, that's just formatting :(

Reply:
well, yea, consisting formatting is nice, isn't it? :)